litellm/router.py

```diff
@@ -17,6 +17,8 @@
 import logging, asyncio
 import inspect, concurrent
 from openai import AsyncOpenAI
+import vertexai.preview.generative_models as vertexai_generative_models
```

Comment:
@arnaud-secondlayer this would require users who aren't using vertex ai to have it's dependencies installed, can we move this import inside your if condition below?

Reply:
Done